### ISSUE TITLE
Enhancement:  Watch/Alert list

### ISSUE THREAD

**Issue by 3vi1** [Xanthos-Eve/vintel#103]
This is a pretty simple one that I think could be really useful.  I'll implement it when I get the time if you agree.  Here goes:

Allow the user to specify a list  that would consist of entries that contain channel, regular expression, and sound file.  Than, whenever that regular expression is seen in that channel, it would play that sound.  For instance:

Corp / >.*(K|k)hasm / someones_saying_my_name.wav
Local / > .* (red|Red|RED)$ / someone_says_red_is_in_system.wav
Corp /  > ((D|d)ock|DOCK) ?(up|UP|now| NOW)? / friends_say_to_dock_immediately.wav

Using regular expressions, it could be incredibly powerful and allow fleets to give auditory alerts to their participants based on shared lists.

**Comment by Xanthos-Eve** [comment]
Good idea though we should avoid scanning corp or local to manage performance. Corp and local get too busy. That said I think we can get plenty of mileage out of this scanning only private intel and fleet channels.

I do like the idea of user selectable sounds too. Although, once we get firmly onto QT5 and python 3 our options open up for text to speech on all platforms so keep that in mind - we can simply say what we want to say to the user.

I would like to place this feature after the refactor of the UI - which is sorely needed. Let's move the feature discussion to the wiki where we can better flesh out our designs and collaborate. I think we can work better there for the overall design and ideation work. Once that is done we can break it up into individual tickets.

**Comment by 3vi1** [comment]
Sounds good!
